src/middlewared/middlewared/plugins/network.py

```diff
@@ -15,6 +15,8 @@
 from .interface.interface_types import InterfaceType
 from .interface.lag_options import XmitHashChoices, LacpduRateChoices
 
+from os import scandir
```

Comment:
Since this is a builtin module, we need to put the import at the top of the file above the `from .inter*` import lines.